.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -13,14 +13,82 @@
 
 .upload-entities-to-cortex-xsoar: &upload-entities-to-cortex-xsoar
   - section_start "Upload Entities to Cortex XSOAR" --collapsed
-  - demisto-sdk upload -i Packs/HelloWorld/Integrations/ --insecure
   - demisto-sdk upload -i Packs/HelloWorld/TestPlaybooks/playbook-HelloWorld-Test.yml --insecure
-  - demisto-sdk upload -i Packs/HelloWorld/Layouts/layoutscontainer-Hello_World_Test_Layout.json --insecure
   - demisto-sdk upload -i Packs/HelloWorld/IncidentFields/incidentfield-Hello_World_IncidentField_Test.json --insecure
   - demisto-sdk upload -i Packs/HelloWorld/IncidentTypes/incidenttype-Hello_World_Alert_Test.json --insecure
-  - demisto-sdk upload -i Packs/HelloWorld/Classifiers/classifier-mapper-incoming-HelloWorldTest.json --insecure
   - section_end "Upload Entities to Cortex XSOAR"
 
+
+.export_cloud_machine_constants: &export_cloud_machine_constants
+  - CLOUD_SERVERS_PATH=$(cat $CLOUD_SERVERS_FILE)
+  - |
+    if [[ $TEST_XDR_ENV ]]; then
+      cat "${CLOUD_API_KEYS}" > "cloud_api_keys.json"
+    else
+      echo "${CLOUD_API_KEYS}" > "cloud_api_keys.json"
+    fi
+  - |
+    if [[ $TEST_XDR_ENV ]]; then
+      cat "${CLOUD_API_TOKENS}" > "cloud_api_token.json"
+    else
+      echo "${CLOUD_API_TOKENS}" > "cloud_api_token.json"
+    fi
+
+  - IFS=', ' read -r -a CLOUD_CHOSEN_MACHINE_ID_ARRAY <<< "${CLOUD_CHOSEN_MACHINE_IDS}"
+  - |
+    for CLOUD_CHOSEN_MACHINE_ID in "${CLOUD_CHOSEN_MACHINE_ID_ARRAY[@]}"; do
+      export XSIAM_SERVER_CONFIG=$(jq -r ".[\"${CLOUD_CHOSEN_MACHINE_ID}\"]" < "$CLOUD_SERVERS_PATH")
+      export DEMISTO_BASE_URL=$(echo "$XSIAM_SERVER_CONFIG" | jq -r ".[\"base_url\"]")
+      export XSIAM_AUTH_ID=$(echo "$XSIAM_SERVER_CONFIG" | jq -r ".[\"x-xdr-auth-id\"]")
+      export DEMISTO_API_KEY=$(jq -r ".[\"${CLOUD_CHOSEN_MACHINE_ID}\"]" < "cloud_api_keys.json")
+      export XSIAM_TOKEN=$(jq -r ".[\"${CLOUD_CHOSEN_MACHINE_ID}\"]" < "cloud_api_tokens.json")
+    done
+
+.run_test_content: &run_test_content
```

Comment:
are there differences between this and the one we have in the on push? looks the same, it is not possible to just call the one that is the on push file?

Reply:
I forgot to remove it, it is the same haha